WordPress/Classes/Services/BlogService.m

```diff
@@ -288,31 +295,68 @@ - (void)syncBlog:(Blog *)blog
 
 - (void)syncBlogStaggeringRequests:(Blog *)blog
 {
-    __weak __typeof(self) weakSelf = self;
+    [self staggerSyncPostsForBlog:blog];
+}
 
+- (void)staggerSyncPostsForBlog:(Blog *)blog
+{
     PostService *postService = [[PostService alloc] initWithManagedObjectContext:self.managedObjectContext];
+    [postService syncPostsOfType:PostServiceTypePost forBlog:blog success:^{
+        [self staggerSyncPagesForBlog:blog];
```

Comment:
No, this is a special case for sites with basic http authentication enabled. We stagger the requests to avoid an issue with failed auth. See #3016.